api/app/hfi/hfi_calc.py

```diff
@@ -26,25 +26,22 @@
                                        get_raw_dailies_in_range_generator)
 from app.db.crud.hfi_calc import (get_fire_centre_stations,
                                   get_fire_weather_stations, get_fire_centre_fire_start_ranges,
-                                  get_fire_start_lookup)
+                                  get_fire_start_lookup, get_fuel_types)
 
 logger = logging.getLogger(__name__)
 
 
-def generate_station_daily(raw_daily,  # pylint: disable=too-many-locals
+def generate_station_daily(raw_daily: dict,  # pylint: disable=too-many-locals
                            station: WFWXWeatherStation,
-                           fuel_type: FuelType) -> StationDaily:
+                           fuel_type: FuelTypeModel) -> StationDaily:
     """ Transform from the raw daily json object returned by wf1, to our daily object.
     """
     # pylint: disable=invalid-name
+    pc = fuel_type.percentage_conifer
+    pdf = fuel_type.percentage_dead_fir
     # we use the fuel type lookup to get default values.
-    pc = fuel_type.percentage_conifer if fuel_type.percentage_conifer is not None\
```

Comment:
Getting rid of defaults here - everything should be coming from the fuel type object.